pose_format/utils/holistic.py

```diff
@@ -15,7 +15,7 @@
 HAND_POINTS = mp_holistic.HandLandmark._member_names_
 HAND_LIMBS = [(int(a), int(b)) for a, b in mp_holistic.HAND_CONNECTIONS]
 
-FACE_POINTS = [str(i) for i in range(468)]
+FACE_POINTS = [str(i) for i in range(468)] # TODO: removed?
```

Comment:
perhaps:
```py
FACE_POINTS_NUM = lambda additional_points=0: additional_points + 468
FACE_POINTS = lambda additional_points=0: [str(i) for i in range(FACE_POINTS_NUM(additional_points))]
```

then this can be used as `FACE_POINTS()` for the default, and `FACE_POINTS(10)` for the extra 10 you use